CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
   ([#261](https://github.com/cucumber/cucumber-ruby-core/pull/261))
 - Permit usage of gherkin v27
 
+## [12.0.1] - 2023-09-17
```

Comment:
Don't make edits to version numbers in PR's. We have an autonomous system that handles all of this for us. Place inside unreleased please.

Reply:
Updated! Thanks @luke-hill !